plugins/evaluation-plugin/src/com/intellij/cce/interpreter/CompletionInvokerImpl.kt

```diff
@@ -210,6 +218,23 @@ class CompletionInvokerImpl(private val project: Project,
     return session
   }
 
+  override fun callImportCompletion(expectedString: String): com.intellij.cce.core.Lookup {
+    val provider = ImportListProvider.EP_NAME.extensions.toList().filter { it.language == language }
```

Comment:
- `toList`is redundunt
- if you expect only one provider, you could use `single` or `singleOrNull` methods instead of `filter`

Reply:
also you could hide extension point in `ImportListProvider` and create companion method `forLanguage(language: Language): ImportListProvider?`